lisa/tools/blobdownload.py

```diff
@@ -0,0 +1,115 @@
+import re
+from pathlib import PurePath
+from typing import Any, List
+
+from lisa.executable import Tool
+from lisa.util import LisaException
+
+from .python import Pip, Python
+
+
+class BlobDownloader(Tool):
```

Comment:
Change current blob download parts to use this tool.

Reply:
there are no places to download blob in current code

Reply:
> there are no places to download blob in current code

what means "no places to download blob?" If you mean the folder to save the downloaded things, you can use the local working folder from local node.

Reply:
Currently there is no existing code to use download blob.